### PR TITLE
Initial commit for the translation of faq-build page

### PR DESCRIPTION
**Pull request by Twaha-Rahman**

### CODE REVIEW

File: content/docs/faq-build.md [hunk 1]
```diff
@@ -6,27 +6,27 @@ layout: docs
...
+না! এখানে দেখুন ["JSX ছাড়া React"](/docs/react-without-jsx.html) আরও জানার জন্য.
 
-### Do I need to use ES6 (+) with React? {#do-i-need-to-use-es6--with-react}
+### আমার কি ES6 (+) ব্যবহার করতে হবে React এর সাথে? {#do-i-need-to-use-es6--with-react}
 
-No! Check out ["React Without ES6"](/docs/react-without-es6.html) to learn more.
+না! এখানে দেখুন ["ES6 ছাড়া React"](/docs/react-without-es6.html) আরও জানার জন্য.
 
-### How can I write comments in JSX? {#how-can-i-write-comments-in-jsx}
+### আমি কিভাবে কমেন্ট লিখব React-এ?{#how-can-i-write-comments-in-jsx}
```
Comment (sazzadsazib) [inline content/docs/faq-build.md]: ### আমি কিভাবে JSX এ কমেন্ট লিখব?{#how-can-i-write-comments-in-jsx}

File: content/docs/faq-build.md [hunk 2]
```diff
@@ -6,27 +6,27 @@ layout: docs
...
 ---
 
-### Do I need to use JSX with React? {#do-i-need-to-use-jsx-with-react}
+### React এর সাথে কি আমার JSX ব্যবহার করতে হবে? {#do-i-need-to-use-jsx-with-react}
 
-No! Check out ["React Without JSX"](/docs/react-without-jsx.html) to learn more.
+না! এখানে দেখুন ["JSX ছাড়া React"](/docs/react-without-jsx.html) আরও জানার জন্য.
 
-### Do I need to use ES6 (+) with React? {#do-i-need-to-use-es6--with-react}
+### আমার কি ES6 (+) ব্যবহার করতে হবে React এর সাথে? {#do-i-need-to-use-es6--with-react}
```
Comment (sazzadsazib) [inline content/docs/faq-build.md]: how about, আমার কি React এর সাথে ES6 (+) ব্যবহার করতে হবে?

File: content/docs/faq-build.md [hunk 3]
```diff
@@ -6,27 +6,27 @@ layout: docs
 category: FAQ
 ---
 
-### Do I need to use JSX with React? {#do-i-need-to-use-jsx-with-react}
+### React এর সাথে কি আমার JSX ব্যবহার করতে হবে? {#do-i-need-to-use-jsx-with-react}
 
-No! Check out ["React Without JSX"](/docs/react-without-jsx.html) to learn more.
+না! এখানে দেখুন ["JSX ছাড়া React"](/docs/react-without-jsx.html) আরও জানার জন্য.
```
Comment (sazzadsazib) [inline content/docs/faq-build.md]: না! আরও জানার জন্য এখানে দেখুন ["JSX ছাড়া React"](/docs/react-without-jsx.html)

**Comment by Twaha-Rahman** [comment]
Made commits to reflect the suggestions made by @shakib609 and @sazzadsazib

File: content/docs/faq-build.md (hunk 1 above)
Comment (Twaha-Rahman) [reply content/docs/faq-build.md]: Yeah, I've made a commit addressing this problem. Thanks for helping!😊

File: content/docs/faq-build.md [hunk 4]
```diff
@@ -6,27 +6,27 @@ layout: docs
 category: FAQ
 ---
 
-### Do I need to use JSX with React? {#do-i-need-to-use-jsx-with-react}
+### React এর সাথে কি আমার JSX ব্যবহার করতে হবে? {#do-i-need-to-use-jsx-with-react}
 
-No! Check out ["React Without JSX"](/docs/react-without-jsx.html) to learn more.
+না! আরও জানার জন্য এখানে দেখুন ["JSX ছাড়া React"](/docs/react-without-jsx.html)।
```
Comment (nutboltu) [inline content/docs/faq-build.md]:
```suggestion
না! বিস্তারিত জানতে ["JSX ছাড়া React"](/docs/react-without-jsx.html) দেখুন।
```

File: content/docs/faq-build.md [hunk 5]
```diff
@@ -6,27 +6,27 @@ layout: docs
...
+  {/* এখানে কমেন্ট হবে */}
   Hello, {name}!
 </div>
 ```
 
 ```jsx
 <div>
-  {/* It also works 
-  for multi-line comments. */}
+  {/* Multi-line কমেন্টের জন্যও
```
Comment (nutboltu) [inline content/docs/faq-build.md]: Mutli-line -> একাধিক লাইনের

File: content/docs/faq-build.md [hunk 6]
```diff
@@ -6,27 +6,27 @@ layout: docs
...
+### React এর সাথে কি আমার JSX ব্যবহার করতে হবে? {#do-i-need-to-use-jsx-with-react}
 
-No! Check out ["React Without JSX"](/docs/react-without-jsx.html) to learn more.
+না! আরও জানার জন্য এখানে দেখুন ["JSX ছাড়া React"](/docs/react-without-jsx.html)।
 
-### Do I need to use ES6 (+) with React? {#do-i-need-to-use-es6--with-react}
+### আমার কি React এর সাথে ES6 (+) ব্যবহার করতে হবে? {#do-i-need-to-use-es6--with-react}
 
-No! Check out ["React Without ES6"](/docs/react-without-es6.html) to learn more.
+না! আরও জানার জন্য এখানে দেখুন ["ES6 ছাড়া React"](/docs/react-without-es6.html)।
```
Comment (nutboltu) [inline content/docs/faq-build.md]:
```suggestion
না! বিস্তারিত জানতে  ["ES6 ছাড়া React"](/docs/react-without-es6.html) দেখুন।
```

**Comment by Twaha-Rahman** [comment]
@nutboltu Committed all the suggestions.

File: content/docs/faq-build.md [hunk 7]
```diff
@@ -6,27 +6,28 @@ layout: docs
...
 </div>
 ```
 
 ```jsx
 <div>
-  {/* It also works 
-  for multi-line comments. */}
+  {/* একাধিক লাইনের
+  কমেন্টের জন্যও
+  এটি কাজ করে */}
```
Comment (nutboltu) [inline content/docs/faq-build.md]: Minor change. please make this comment into 2 lines as the original

File: content/docs/faq-build.md [hunk 8]
```diff
@@ -6,27 +6,28 @@ layout: docs
...
 ---
 
-### Do I need to use JSX with React? {#do-i-need-to-use-jsx-with-react}
+### React এর সাথে কি আমার JSX ব্যবহার করতে হবে? {#do-i-need-to-use-jsx-with-react}
 
-No! Check out ["React Without JSX"](/docs/react-without-jsx.html) to learn more.
+না! বিস্তারিত জানতে ["JSX ছাড়া React"](/docs/react-without-jsx.html) দেখুন।
 
-### Do I need to use ES6 (+) with React? {#do-i-need-to-use-es6--with-react}
+### আমার কি React এর সাথে ES6 (+) ব্যবহার করতে হবে? {#do-i-need-to-use-es6--with-react}
```
Comment (nutboltu) [inline content/docs/faq-build.md]: Let's keep it consistent as previous question.
```suggestion
### React এর সাথে কি আমার ES6 (+) ব্যবহার করতে হবে? {#do-i-need-to-use-es6--with-react}
```

**Comment by Twaha-Rahman** [comment]
@nutboltu Committed **all** the suggestions.

**Comment by Twaha-Rahman** [comment]
@nutboltu Well....we have little bit of problem here😅. I think I forgot to translate **title: Babel, JSX and Build Steps** -> **title: Babel, JSX এবং Build Steps**

**Pull request merged**